lisa/features/gpu.py

```diff
@@ -3,45 +3,61 @@
 
 import re
 from enum import Enum
-from typing import Any
+from typing import Any, List, Set
 
-from lisa.base_tools.wget import Wget
+from lisa.base_tools import Uname, Wget
 from lisa.feature import Feature
 from lisa.operating_system import Redhat, Ubuntu
-from lisa.tools import Uname
+from lisa.sut_orchestrator.azure.tools import LisDriver
+from lisa.tools import Lspci, Lsvmbus
 from lisa.util import LisaException, SkippedException
 from lisa.util.logger import get_logger
 
 FEATURE_NAME_GPU = "Gpu"
 
-ComputeSDK = Enum(
-    "ComputeSDK",
-    [
-        # GRID Driver
-        "GRID",
-        # CUDA Driver
-        "CUDA",
-    ],
-)
-
 # Link to the latest GRID driver
 # The DIR link is
 # https://download.microsoft.com/download/9/5/c/95c667ff-ab95-4c56-89e0-e13e9a76782d/NVIDIA-Linux-x86_64-460.32.03-grid-azure.run
 DEFAULT_GRID_DRIVER_URL = "https://go.microsoft.com/fwlink/?linkid=874272"
 
 
 class Gpu(Feature):
+
+    ComputeSDK = Enum(
+        "ComputeSDK",
+        [
+            # GRID Driver
+            "GRID",
+            # CUDA Driver
+            "CUDA",
+        ],
+    )
+
     def __init__(self, node: Any, platform: Any) -> None:
         super().__init__(node, platform)
         self._log = get_logger("feature", self.name(), self._node.log)
+        self.gpu_vendor: Set[str]
+        self.gpu_name = ["Tesla", "A100-SXM4"]
```

Comment:
Add comments above to explain it. And because it's associate with device id, it can be a mapping or tuple. It's easier to use them together, and easy to extend.